src/promptflow-devkit/promptflow/_cli/pf.py

```diff
@@ -1,8 +1,17 @@
+#!/usr/bin/env python
 # ---------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
+import os
+import sys
+
+
+def main():
+    if os.environ.get("PF_INSTALLER") is None:
+        os.environ["PF_INSTALLER"] = "PIP"
+
+    os.execl(sys.executable, sys.executable, "-m", "promptflow._cli._pf.entry", *sys.argv[1:])
```

Comment:
Not sure whether it's my local dev environment issue, it seems strange when I run pf command in the branch.

![image](https://github.com/microsoft/promptflow/assets/26239730/7876f688-69ba-46cb-bbbb-18d946886889)